module/VuFind/src/VuFind/Auth/CAS.php

```diff
@@ -286,18 +287,28 @@ protected function setupCAS()
         // client can only be called once.
         if (!$this->phpCASSetup) {
             $cas = $this->getConfig()->CAS;
-            if (
-                isset($cas->log)
-                && !empty($cas->log) && isset($cas->debug) && ($cas->debug)
-            ) {
-                $casauth->setDebug($cas->log);
+            $casauth->setLogger(new PsrLoggerAdapter($this->logger));
+            if (isset($cas->debug) && ($cas->debug)) {
+                $casauth->setVerbose(true);
             }
             $protocol = constant($cas->protocol ?? 'SAML_VERSION_1_1');
+            $service_base_url = null;
```

Comment:
As a minor point of style, I'd rename this variable to `$serviceBaseUrl` since we generally use camelCase instead of snake_case for PHP variables.